### PR TITLE
Added more details on Azure DNS challenge

### PR DESCRIPTION
**Pull request by samanamp**
Changed the docs>configuration/acme/dns01/azuredns to have more details on how to configure the system. Changed the order of a paragraph, added comments on scripts where needed and omitted where the reader can imply easily. Changed variable names to reflect what is present on Azure's Portal.

/assign @joshvanl

### CODE REVIEW

File: content/en/docs/configuration/acme/dns01/azuredns.md [hunk 1]
```diff
@@ -8,33 +8,37 @@ type: "docs"
...
 To create the service principal you can use the following script (requires
 `azure-cli` and `jq`):
 
 ```bash
-$ AZURE_CERT_MANAGER_SP_NAME=SOME_SERVICE_PRINCIPAL_NAME
-$ AZURE_CERT_MANAGER_DNS_RESOURCE_GROUP=SOME_RESOURCE_GROUP
-$ AZURE_CERT_MANAGER_DNS_NAME=SOME_DNS_ZONE
+# Choose a name for the service principal that contacts azure DNS to present the challenge
+$ AZURE_CERT_MANAGER_NEW_SP_NAME=NEW_SERVICE_PRINCIPAL_NAME
+# This is the name of the resource groupe that you have your dns zone in it
```
Comment (JoshVanL) [inline content/en/docs/configuration/acme/dns01/azuredns.md]: groupe -> group

remove 'it'?

**Comment by munnerz** [comment]
@samanamp any chance you can rebase this PR (there was a minor change made in #96) and address the comments above so we can get this in? It's great to have documentation changes like this as they usually really help end-users getting started for the first time!

**Comment by munnerz** [comment]
Ping @samanamp - do you think you'll have some time to follow up on this?

**Comment by samanamp** [comment]
@munnerz Will look into rebasing it soon. Sorry for the delay. :)

**Comment by maelvls** [comment]
Hi!

The PR seems to be 99% done and reviewed.

@samanamp Would you prefer to do the rebase and PR comment fixes, or would you prefer I do the changes?

**Pull request closed without merging**